doc/release/upcoming_changes/26901.changes.rst

```diff
@@ -0,0 +1 @@
+* `numpy.loadtxt` no longer set `F_CONTIGUOUS` for text files with multiple rows and large row sizes.
```

Comment:
This file isn't named correctly, I think it's `.change.`.  The change is a small/bug fix enough that we don't need it (but also OK to keep it).

We also have a Changelog, that just lists all PR titles, so not having a fragment here doesn't mean it's lost completely.